portfolio/src/main/java/com/google/sps/servlets/OfficeGame.java

```diff
@@ -55,7 +61,20 @@
   /** Is true or false based on whether the user got the last question right */
   private boolean lastSubmission = true;
 
-  private int score = 0;
+  private long score = 0;
+
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  private long highScore = -1;
+
+  public OfficeGame(){
+    Query query = new Query("Score").addSort("score", SortDirection.DESCENDING);
+    PreparedQuery results = datastore.prepare(query);
+    for(Entity entity : results.asIterable()){
```

Comment:
I think data store is not needed here (see comment below). But in the implementation here, you probably only want the first entry of your `results` since that's the all time highest score.

Reply:
I changed it so that it only accesses and changes the first entry, it never creates a new one so we do not have a bunch of newly created entities everytime the high score is increased